### PR TITLE
Added OWNERDOMAIN & MANAGERDOMAIN

### LINKED ISSUE #107: Missing variables  OWNERDOMAIN and MANAGERDOMAIN
### Is your enhancement related to a problem? Please describe.

https://iabtechlab.com/wp-content/uploads/2022/04/Ads.txt-1.1.pdf adds support for OWNERDOMAIN and MANAGERDOMAIN which are currently still throwing errors in the plugin and get reported as "Unrecognized variable"

It would be also good to add a filter to overwrite the default set if needed. 

### Designs

_No response_

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by SoftCreatR**
Resolves #107

### Changelog Entry

> Added - Support for OWNERDOMAIN & MANAGERDOMAIN per version 1.1 of the spec.

### Credits

Props @SoftCreatR, @tott.

### CODE REVIEW

**Comment by jeffpaul** [comment]
@SoftCreatR welcome to Ads.txt Manager and thanks for the PR.  I've passed this along to our team to review and will get back to you with any concerns... cheers!

File: inc/save.php [hunk 1]
```diff
@@ -105,7 +105,7 @@ function validate_line( $line, $line_number ) {
 		$sanitized = wp_strip_all_tags( $line );
 	} elseif ( 1 < strpos( $line, '=' ) ) { // This is a variable declaration.
 		// The spec currently supports CONTACT, INVENTORYPARTNERDOMAIN and SUBDOMAIN.
```
Comment (peterwilsoncc) [inline inc/save.php]:
```suggestion
		// The spec currently supports CONTACT, INVENTORYPARTNERDOMAIN, SUBDOMAIN, OWNERDOMAIN and MANAGERDOMAIN.
```

**Review by peterwilsoncc — COMMENTED** [review]
This looks good and works using the test code from the spec.

I've added a suggestion inline to update the comment above the change. I'll self merge that so we can merge the pull request.

Thanks for your help maintaining ads.txt!

**Comment by SoftCreatR** [comment]
Dang, I've missed that :)

**Review by peterwilsoncc — APPROVED** [review]

**Comment by peterwilsoncc** [comment]
It's easily done :) 

I've approved the PR but noticed the repo's settings need to be updated before I can merge. Once that happens in the next few days, I'll merge.

**Pull request merged**